internal/config/aws.go

```diff
@@ -0,0 +1,55 @@
+/**
+* Copyright 2023-present Coinbase Global, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*  http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package config
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awsConfig "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+)
+
+func (app AppConfig) GenerateAwsConfig() aws.Config {
+	var cfg aws.Config
+	var err error
+	if app.Env == "awslocal" {
+		cfg, err = awsConfig.LoadDefaultConfig(context.Background(),
+			awsConfig.WithSharedConfigProfile("sa-infra"),
```

Comment:
What does sa-infra map to? Need an env var?

Reply:
yeah that's just my AWS profile i've been using. I'll map it into an env var